src/katgpucbf/xbgpu/engine.py

```diff
@@ -1026,7 +1026,7 @@ class XBEngine(DeviceServer):
         The number of consecutive batches to store in the same chunk. The higher
         this value is, the more GPU and system RAM is allocated, the lower,
         the more work the Python processing thread is required to do.
-    rx_reorder_tol
+    recv_reorder_tol
```

Comment:
I see we've got further inconsistency, in that parameters also get named `src` and `dst` rather than `recv` and `send`. In the longer term we should unify that; in the shorter term I'd suggest it would be more locally consistent to name this `src_reorder_tol` to match the other `src_*` parameters.